jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt

```diff
@@ -65,30 +71,21 @@ class SchemaSelectionPanelTest {
     private val CUSTOMER_UPLOADED_SCHEMA_MULTIPLE_TYPES =
         File(javaClass.getResource("/customerUploadedEventSchemaMultipleTypes.json.txt").toURI()).readText(Charsets.UTF_8)
 
-    private val mockSamProjectBuilder = mock<SamProjectBuilder>()
-
     private val mockResourceSelector = mock<ResourceSelector<SchemaSelectionItem>>()
-    private val mockPanel = mock<JPanel>()
 
     private val mockResourceBuilderOptions = mock<ResourceSelector.ResourceBuilderOptions<SchemaSelectionItem>>()
     private val mockResourceSelectorBuilder = mock<ResourceSelector.ResourceBuilder>()
 
-    private lateinit var schemaSelectionPanel: SchemaResourceSelectorSelectionPanel
+    private lateinit var schemaSelectionPanel: SchemaResourceSelector
 
     @Before
     fun setUp() {
         initMockResourceCache()
-        initMockResourceSelector()
+//        initMockResourceSelector()
```

Comment:
line commented out, should it be there still?

Reply:
This test is all `@Ignore` and needs to probably be re-written from ground up